src/panel/direction/DirectionForm.jsx

```diff
@@ -89,6 +89,7 @@ export default class DirectionForm extends React.Component {
         </div>
 
         <button
+          disabled={originInputText === '' && destinationInputText === ''}
```

Comment:
Great for the disabled! It's far more consistent like this.
And now it really looks like it should be a standard <Button> component :) It will just need to add `disabled` as a prop to it.

Reply:
Disabled props should now be properly handled by the Button component :record_button: